src/components/AddDoctor/AddDoctor.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Container } from 'react-bootstrap';
+import CreateDoctorForm from './DoctorForm';
+import Response from './Response';
+
+function AddDoctor() {
```

Comment:
- [ ] OPTIONAL: In the 'Add doctor' page, there is no way to go back, even after adding a doctor:

![image](https://user-images.githubusercontent.com/57421823/234658011-19d3501f-d6fa-40f6-a0c0-64f8aa597495.png)

I suggest implementing a "back" button just as you did in the other sections, or making the hamburger button visible to show the menu. 👍 

_note:  Deeming this as optional because it is still possible to press the back button in the browser, however implementing the back button or showing the hamburger will hugely improve the UX. 💪_